cleanlab/internal/neighbor/types.py

```diff
@@ -0,0 +1,17 @@
+from typing import Callable, Union
```

Comment:
Can you move these definitions to the existing file here?

https://github.com/cleanlab/cleanlab/blob/master/cleanlab/typing.py